### PR TITLE
addressed review comments

### PR DESCRIPTION
**Pull request by wdwatkins**

### CODE REVIEW

**Comment by lindsaycarr** [comment]
:+1:

File: content/statsServiceMap.md [hunk 1]
```diff
@@ -7,7 +7,6 @@ title: Using the dataRetrieval Stats Service
 categories: Data Science
 tags: 
   - R
-  - dataRetrieval
```
Comment (ldecicco-USGS) [inline content/statsServiceMap.md]: I know it's a pain, but we do need to manually enter anything past R...and I'm a big fan of keeping "dataRetrieval" as a tag here

Comment (lindsaycarr) [inline content/statsServiceMap.md]: me too! I breezed right by that. How did you set it up to automatically add R?

Comment (ldecicco-USGS) [inline content/statsServiceMap.md]: I modified the package to always add R....but I didn't figure out a way to add an argument that the user could define(ie...dataRetrieval or geoknife)...if someone wants to mess with the package, that would be great:

https://github.com/USGS-R/USGSmarkdowntemplates

Comment (wdwatkins) [inline content/statsServiceMap.md]: Yeah forgot about adding that back.  An automatic or user-defined 'image' header field would be nice too.

**Comment by wdwatkins** [comment]
@ldecicco-USGS PR'd to `USGSmarkdowntemplates`.  It's pretty easy to add fields.

**Pull request merged**